### PR TITLE
Update V1 Install docs

### PR DESCRIPTION
**Pull request by pragnagopa**
Fixes https://github.com/kedacore/keda/issues/1657

### CODE REVIEW

**Comment by pragnagopa** [comment]
cc @tomkerkhove

File: content/docs/1.4/deploy.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ kubectl apply -f ./
 You can also find the same YAML declarations in our `/deploy` directory on our [GitHub repo](https://github.com/kedacore/keda) if you prefer to clone it.
 
 ```sh
-git clone https://github.com/kedacore/keda && cd keda
+git clone https://github.com/kedacore/keda && cd keda && git checkout v1
```
Comment (tomkerkhove) [inline content/docs/1.4/deploy.md:97]: Might be better to target our tag maybe?
```suggestion
git clone https://github.com/kedacore/keda && cd keda && git checkout tags/v1.4.1
```

**Review by tomkerkhove — COMMENTED** [review]
Thanks, just a small suggestion and was wondering if you would be able to do the same for v1.5?

**Review by tomkerkhove — APPROVED** [review]
Thanks!

**Pull request merged**